views/beneficial-owner-type.html

```diff
@@ -87,37 +90,24 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ boTypesTitle }}</h1>
           items: boTypes
         }) }}
 
-        {{ govukButton({
-          text: "Add",
-          attributes: {
-            "id": "add"
-          }
-        }) }}
+        <button class="govuk-button" data-module="govuk-button" id="add">Add <span class="govuk-visually-hidden"> {{ whatToAdd }} </span></button>
```

Comment:
the examples James Francis gives look like they are taken from the page source rather than the nunjucks components. Does it work the same if you add a govuk-visually-hidden attribute to the govukButton component?

Reply:
The problem is that the govuk-visually-hidden attribute is attached to a nested element not directly to the button.

Reply:
ok, just wondered if there was an easier way to do it with nunjucks. Stick with the html buttons.